Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -58,6 +69,13 @@ module.default = class ZipScript extends QuickScript {
             .to(wine.prefixDirectory() + "/drive_c/" + this._name)
```

Comment:
This is a bit nitpicking I know but I believe:

```javascript
.to(`${wine.prefixDirectory()}/drive_c/${this._name}`)
```

to be much easier readable

I also think this about the other path strings in this file

Reply:
After this has been fixed I think we can merge this :)